### PR TITLE
Updating the readme for Windows 10 support.

### PR DESCRIPTION
**Pull request by Jayflux**

### CODE REVIEW

**Comment by Jayflux** [comment]
@bnoordhuis

File: README.md [hunk 1]
```diff
@@ -50,6 +50,14 @@ You will also need to install:
         * If you get errors that the 64-bit compilers are not installed you may also need the [compiler update for the Windows SDK 7.1]
     * Windows 7/8:
       * Microsoft Visual Studio C++ 2013 for Windows Desktop ([Express][msvc2013] version works well)
+    * Windows 10:
+        * Install the latest version of npm (3.3.6 at the time of writing)
+        * Install Python 2.7 from Python 2.7 from https://www.python.org/download/releases/2.7/ and make sure its on the System Path
```
Comment (TooTallNate) [inline README.md]: Remove one instance of `Python 2.7 from`

**Comment by Jayflux** [comment]
@TooTallNate I've updated the branch with those changes but weirdly it doesn't show here
#rebaseNewbie

**Comment by noamokman** [comment]
Hi @Jayflux! looks like you accidentally created two branches :sweat_smile: 
Your PR is under [upatingWindows10Doc](https://github.com/Jayflux/node-gyp/tree/upatingWindows10Doc) which has a typo.
Your fix is on a different branch: [updatingWindows10Doc](https://github.com/Jayflux/node-gyp/tree/updatingWindows10Doc).

Good luck!

**Comment by Jayflux** [comment]
hah yes @noamokman 
If @TooTallNate  wants to merge in updatingWindows10Doc and delete this one that would be great

**Comment by saper** [comment]
@jayflux why npm 3.3.6 is required on Windows 10? Python installation is already covered.

Is VS2015 not working? I think VS2013 would work too?

**Comment by bnoordhuis** [comment]
The documentation links to Windows Build Tools now so that should cover Windows 10 as well.  I'll go ahead and close but thanks for taking the time to put together this pull request.

**Pull request closed without merging**